modules/utxo-lib/src/bitgo/outputScripts.ts

```diff
@@ -115,25 +116,47 @@ export function createOutputScript2of3(pubkeys: Buffer[], scriptType: ScriptType
  * @param pubkeys - a pubkey array containing the user key, backup key, and bitgo key in that order
  * @returns {{scriptPubKey}}
  */
-function createTaprootScript2of3([userKey, backupKey, bitGoKey]: Triple<Buffer>): SpendableScript {
-  const userBitGoScript = bitcoinjs.script.compile([userKey, OP_CHECKSIGVERIFY, bitGoKey, OP_CHECKSIG]);
-  const userBackupScript = bitcoinjs.script.compile([userKey, OP_CHECKSIGVERIFY, backupKey, OP_CHECKSIG]);
-  const backupBitGoScript = bitcoinjs.script.compile([backupKey, OP_CHECKSIGVERIFY, bitGoKey, OP_CHECKSIG]);
-
-  assert(userBitGoScript);
-  assert(userBackupScript);
-  assert(backupBitGoScript);
-
-  const { output } = bitcoinjs.payments.p2tr({
-    pubkeys: [userKey, bitGoKey],
-    scripts: [userBitGoScript, userBackupScript, backupBitGoScript],
-    weights: [2, 1, 1],
+function createTaprootScript2of3([userKey, backupKey, bitGoKey]: Triple<Buffer>, redeemIndex = 0): bitcoinjs.Payment {
+  const keyCombinations2of2 = [
+    [userKey, bitGoKey],
+    [userKey, backupKey],
+    [backupKey, bitGoKey],
+  ].map(([key1, key2]) => [key1.slice(1), key2.slice(1)]); // remove the y odd/even prefix byte
+  const redeems = keyCombinations2of2.map((pubkeys, index) => 
+    bitcoinjs.payments.p2tr_ns({
+      pubkeys,
+      weight: index === 0 ? 2 : 1,
+    })
+  );
+
+  return bitcoinjs.payments.p2tr({
+    pubkeys: keyCombinations2of2[0],
+    redeems,
+    redeemIndex,
   });
+}
 
+/**
+ * Returns the conrtrol block and tapscript necessary for creating a witness stack for a script
+ * path spend using one of the three n of n scripts in each taproot's taptree.
+ * @param pubkeys - a pubkey array containing the user key, backup key, and bitgo key in that order
+ * @param redeemIndex 
+ * @returns 
+ */
+export function prepareP2trP2nsWitness(pubkeys: Triple<Buffer>, redeemIndex = 0) {
+  const {
+    controlBlock,
+    redeem,
+    output,
```

Comment:
`output` won't be necessary if/when `TransactionBuilder` derives it on its own from the control block and tapscript.